templates/page/layout/municipales.html.twig

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html.twig' %}
+
+{% block page_title page.title %}
+{% block canonical_url url('app_static_page', { slug: page.slug }) %}
+
+{% block header '' %}
+
+{% block content %}
+    <main style="background-color: red;">
```

Comment:
yes. i know. it was my live test.

@enzozccr you can now work on this first new layout `municipales` :tada:

Reply:
thank you my man